### PR TITLE
Adding a back button for the sidebar and more 

### LINKED ISSUE #744: [Feature Request]:  Update button click event in app.js and add new button in index.html
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

I am adding back button in sidebar and adding some details in the detail section , and also changing the app.js section for the back button.

### Use Case

The back button will help in closing the sidebar when not in use and the additional detail will help in details with the company. 

### Benefits

_No response_

### Add ScreenShots

This is the website before changes
Detail
Sidebar
This is after the changes 
updatedDetail
updatedSidebar

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Suvendu-UI**
# Title and Issue number 

Title :Adding a back button for the sidebar and more and adding more info for the detail section in the company section 

Issue No. : #744

Code Stack : HTNL, CSS, JS

Close #744

# Video (mandatory)

This is the website before changes

https://github.com/apu52/METAVERSE/assets/144688422/5733cfab-b708-447b-853d-0dfd86244906

This is the website after changes to the website

https://github.com/apu52/METAVERSE/assets/144688422/d91000b4-4fc3-4dc9-8f7a-753a422e1396

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have gone through the  `contributing.md` file before contributing

**Additional context (Mandatory )**

***Are you contributing under any Open-source programme?***

I am contributing under GSSoC 2024.

### CODE REVIEW

File: Projects/Career-Crafters/app.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ $(".menu-bar").on("click",function(){
     $(".sidebar").addClass("active")
 });
 
-$(".logo").on("click",function(){
+$(".btn").on("click",function(){
```
Comment (apu52) [inline Projects/Career-Crafters/app.js:13]: image

it is not working .Please try to recheck the functions u have add

**Review by apu52 — CHANGES_REQUESTED** [review]

**Comment by Suvendu-UI** [comment]
That back button will only work if the hamburger is required to open the sidebar or else it will be static.

File: Projects/Career-Crafters/app.js (hunk 1 above)
Comment (Suvendu-UI) [reply Projects/Career-Crafters/app.js:13]: That back button will only work if the hamburger is required to open the sidebar or else it will be static.

**Comment by apu52** [comment]
> That back button will only work if the hamburger is required to open the sidebar or else it will be static.

okk

**Review by apu52 — APPROVED** [review]

**Pull request merged**